pudb/var_view.py

```diff
@@ -285,15 +287,15 @@ def type_stringifier(value):
         try:
             return text_type(value)
         except Exception:
-            pass
+            ui_log.exception('string safe type stringifier failed')
 
     elif hasattr(type(value), "safely_stringify_for_pudb"):
         try:
             # (E.g.) Mock objects will pretend to have this
             # and return nonsense.
             result = value.safely_stringify_for_pudb()
         except Exception:
-            pass
+            ui_log.exception('safely_stringify_for_pudb call failed')
```

Comment:
Perhaps update `result` to reflect a brief version of the error? (And point to the internal shell for traceback?)